setup.cfg

```
@@ -100,35 +101,37 @@ http2 =
 contextvars =
     contextvars >= 2.4, < 3; python_version < "3.7"
 
-all_non_platform =
-    %(test)s
+all =
+    pyobjc-core; platform_system == "Darwin"
+    pyobjc-framework-CFNetwork; platform_system == "Darwin"
+    pyobjc-framework-Cocoa; platform_system == "Darwin"
+    pywin32 != 226; platform_system == "Windows"
     %(tls)s
     %(conch)s
     %(serial)s
     %(http2)s
     %(contextvars)s
 
+all_non_platform =
+    %(all)s
```

To keep backward compatiblity

```suggestion
all_non_platform =
    %(test)s
    %(all)s
```

---

Thanks for the clarification, I've updated the patches.